src/components/Modal/BaseModal.js

```diff
@@ -77,6 +77,7 @@ class BaseModal extends PureComponent {
             },
             this.props.popoverAnchorPosition,
             this.props.containerStyle,
+            this.props.extraModalStyles,
```

Comment:
I don't really like the name `extraModalStyles`, can we come up with something a bit more descriptive?

Reply:
Thanks for your patience guys have been out sick.

I like `outerStyle`? And I think we can rename `containerStyle` to `innerContainerStyle` since the container those styles are being applied to lives inside the modal.

Reply:
@Ollyws yeah go ahead! Sorry for the delay